js/styles/_typography.scss

```diff
@@ -1,7 +1,35 @@
 // Mixins taken from node_modules/govuk-frontend/govuk/core/_typography
-@import "node_modules/govuk-frontend/govuk/base";
+@import "node_modules/govuk-frontend/dist/govuk/base";
 
-$govuk-typography-use-rem: false;
```

Comment:
we should remove all the extra styles in here, it should just be % prefixed bits- and only the heading-m and body-m

if we don't we'll end up changing styles on the page (possibly)